docs/decisions/0001-connexion-replacement.md

```diff
@@ -0,0 +1,602 @@
+# Connexion Alternatives Decision Record
+
+* Status: Proposed
+* Deciders: TODO
+* Date: 2022-09-27
+
+Technical Story: https://github.com/navapbc/template-application-flask/issues/12
+
+## Context and Problem Statement
+
+This template library was built out using Flask + Connexion as the API technology. We are investigating if we want to keep using Connexion, or potentially move to another Flask-wrapper API approach. Any approach we consider must make it possible to easily setup an OpenAPI/Swagger endpoint.
+
+We aren't looking to move off of Flask at the moment as we feel that would be a much larger refactor.
+
+## Decision Drivers
```

Comment:
I stumbled across [this article by Swagger itself about design-first vs code-first approaches](https://swagger.io/blog/api-design/design-first-or-code-first-api-development/) and think it might be nice to include in the ADR. Perhaps we can reference it and talk about how in most of our projects we default to speed of development early on in the project, which is why we want something like that for the template project, and that it still allows projects to individually migrate to a design-first approach if they want later on in their project.